parameter_store/models.py

```diff
@@ -180,11 +180,6 @@ class Meta:
         verbose_name_plural = 'Cluster Intent'
 
     cluster = models.OneToOneField(Cluster, on_delete=models.CASCADE, related_name="intent")
-    unique_zone_id = models.CharField(
-        max_length=64,
-        unique=True,
-        verbose_name='Unique Zone ID',
-        help_text='This is the ID that uniquely identifies a zone in the ordering process.')
```

Comment:
this field is meant to be an explicit replacement for zone_name in: 
https://github.com/GDC-ConsumerEdge/automated-cluster-provisioner/blob/main/example-source-of-truth.csv